src/main/java/com/answerdigital/answerking/service/TagService.java

```diff
@@ -118,4 +120,27 @@ private void addProductsToTag(final Tag tag, final List<Long> productIds) {
         productService.validateProductsAreNotRetired(products);
         tag.setProducts(products);
     }
+
+    /**
+     * Retires a Tag {@link com.answerdigital.answerking.model.Tag}.
+     * @param tagId The Tag ID to retire.
+     */
+    public void retireTag(final Long tagId) {
```

Comment:
Could we please refactor this 'tagId' to just 'id'?

Very sorry that this is a nitpicky detail, but it's another thing on my TODO list to make a ticket for renaming these.